examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1071 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            random_gate_set = np.random.choice(choice_gate_set, p=[0.30, 0.70])
</pre><p>Is this doing a weighting of 30/70% between the two options? Why not 50/50?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
We found that if it was 50/50 then the function would create too many constant oracles and we wanted a good mix of both that's why the probability is uneven.